src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -1001,6 +1001,7 @@ namespace EnergyPlus::ZoneEquipmentManager {
                 FinalZoneSizing(CtrlZoneNum).HeatDesTempDiff = ZoneSizingInput(ZoneSizNum).HeatDesTempDiff;
                 FinalZoneSizing(CtrlZoneNum).CoolDesHumRat = ZoneSizingInput(ZoneSizNum).CoolDesHumRat;
                 FinalZoneSizing(CtrlZoneNum).HeatDesHumRat = ZoneSizingInput(ZoneSizNum).HeatDesHumRat;
+                FinalZoneSizing(CtrlZoneNum).ZoneAirDistributionIndex = ZoneSizingInput(ZoneSizNum).ZoneAirDistributionIndex;
```

Comment:
This is needed in order to fetch the DesignSpecification:AirTerminal:Sizing data.